README.md

```diff
@@ -15,61 +21,307 @@ Our other repositories are:
+itself, which can be done with this command:
+
+```sh
+tfenv install latest:^1.4.0
+```
+
+_NOTE: This project currently uses the latest `1.4.x release of Terraform._
 
-### Python dependency management
+#### Python Installation
+
+Now we're going to install a tool to help us manage Python versions on our
+system.  First, we'll install [pyenv](https://github.com/pyenv/pyenv) with
+Homebrew:
+
+```sh
+brew install pyenv
+```
+
+When that finishes installing, you'll need to make another adjustment in the
+file that you adjusted for your `$PATH` environment variable and Homebrew's
+setup. Open the file, and add these lines to it:
+
+```
+# pyenv setup
+export PYENV_ROOT="$HOME/.pyenv"
+command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init -)"
+```
+
+Once again, start a new shell session or source the file in your current shell
+session to make the changes take effect.
+
+Now we're ready to install the Python version we need with `pyenv`, like so:
+
+```sh
+pyenv install 3.9
+```
+
+_NOTE: This project currently runs on Python 3.9.x._
+
+This will install the latest version of Python 3.9.
+
+#### Python Dependency Installation
+
+Lastly, we need to install the tool we use to manage Python dependencies within
+the project, which is [poetry](https://python-poetry.org/).
+
+Visit the
+[official installer instructions page](https://python-poetry.org/docs/#installing-with-the-official-installer)
+and follow the steps to install Poetry directly with the script.
+
+This will ensure `poetry` doesn't conflict with any project virtual environments
+and can update itself properly.
+
+#### PostgreSQL installation
+
+We now need to install a database - this project uses PostgreSQL, and Homebrew
+requires a version number to be included with it when installing it:
+
+_NOTE: This project currently works with PostgreSQL version 15.x_
+
+```sh
+brew install postgresql@15
+```
+
+You'll now need to modify (or create, if it doesn't already exist) the `$PATH`
+environment variable to include the PostgreSQL binaries. Open the file you have
+worked with before to adjust your shell environment with the previous steps and
+do one of the following:
+
+If you already have a line that modifies the `$PATH` environment variable, just
+add this path into the existing string:
+
+```
+/opt/homebrew/opt/postgresql@15/bin
+```
+
+If you don't have a line for your `$PATH` environment variable, add it in like
+this, which will include the PostgreSQL binaries:
+
+```
+export PATH="/opt/homebrew/opt/postgresql@15/bin:$PATH
+```
+
+_NOTE: You don't want to overwrite your existing `$PATH` environment variable! Hence the reason why it is included on the end like this; paths are separated by a colon._
+
+#### Starting PostgreSQL and Redis
+
+With both PostgreSQL and Redis installed, you now need to start the services.
+Run this command so that they're available at all times going forward on your
+machine:
+
+```sh
+brew services start postgresql@15 redis
```

Comment:
For all subcommands, `brew services` requires each service to be acted on separately, so these'll need to be separate lines. There is a `--all` flag for starting all services, but that's probably not good to recommend

Reply:
Thank you for the catch!